Yoast/yoast-components
# Fix snippet preview slug

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where categories weren't properly reflected in the Snippet Preview's slug.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* See https://github.com/Yoast/wordpress-seo/pull/11210 for test instructions.
